### PR TITLE
setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by farouqdaif1**
1. set up linters
2. make Html Page 
3. make CSS Page 
 thanks for your review ❤️

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @farouqdaif1, 

**This is a great project you have going on, well-done lad**. 
   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Farouq Daif</title>
+    <!-- Main Template CSS File -->
+    <link rel="stylesheet" href="/css/style.css">
```
Comment (mke2111) [inline index.html:10]: - [ ] Change the href to `"./css/style.css"`. Without the dot, the css file is not referenced using raw.githack and the local version.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+<body>
+  <header>
+    <nav id="bar-tools">
+        <div id="name">
+          <p>Farouq Daif</p>
+        </div>
+          <div id="burger">      
+              <i class="fas fa-bars"></i>
+          </div>
+    </nav>
```
Comment (mke2111) [inline index.html:28]: - [ ] Only enclose the humburg icon within the `nav` tags and not the entire header.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+          <div class="hed1">
+            <P>Hey There.
+              I’m Farouq</p>
+          </div>
+          <div class="hed2">
+            <h2>I’am a Software Developer</h2>
+          </div>
+          <div class="hed3">
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+              don’t hestiate to contact me.</p>
```
Comment (mke2111) [inline index.html:43]: - [ ] For this section of your page, enclose it within `section` tags instead of `div` tags

File: index.html [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+          </div>
+    </nav>
+</header>
+    <main>
+      <div class="parent">
+        <div class="container">
+          <div id="headline">
+          <div class="hed1">
+            <P>Hey There.
+              I’m Farouq</p>
```
Comment (mke2111) [inline index.html:36]: - [ ] Enclose your main title within `h1` tags instead of `p` tags

File: index.html (hunk 3 above)
Comment (farouqdaif1) [reply index.html:43]: I added the Background Image !!

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+
+html {
+  scroll-behavior: smooth;
+}
+
+.parent {
+  background: #1c1a19;
+  width: 375px;
+  height: 812px;
+}
```
Comment (dasileker) [inline css/style.css:19]: - [ ] The width & height is not correct, please fix it.

kfkfk

File: index.html (hunk 1 above)
Comment (dasileker) [reply index.html:10]: - Like previews TSE said the href link should be written as he mentioned.

File: index.html (hunk 2 above)
Comment (dasileker) [reply index.html:28]: - Also the logo you should follow the previews TSE criteria.

jdjdj

File: index.html (hunk 4 above)
Comment (dasileker) [reply index.html:36]: - Please do the required changes that the  previews TSE sais.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @farouqdaif1,** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DanSam5K — APPROVED** [review]
# Hi @farouqdaif1 ✌️  Approved
I can see you have put in a great among of work and effort trying to fix all the required changes by the previous reviewer good job 👏 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## [Optional] Suggestion
- Since the previous Reviewer forgot to mention, I want to suggest that you try to fix the following it will greatly help improve your work
- [ ] You have two CSS files, I will suggest you remove the one you are not currently using to keep your work clean
- [ ] Also you can improve your README with good documentation by fixing broken screenhshots and Typo

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+  box-sizing: border-box;
+  margin: 0;
+  border: 0;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
```
Comment (SajjadAhmad14) [inline css/style.css:15]: [  OPTIONAL ] The page has a horizontal scroll which means some content is overflowing from the container. Please fix it as it's not considered a good design.
overflow

**Review by SajjadAhmad14 — COMMENTED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Farouq Daif, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏
# REMINDER
### Please do not merge and close PR before it's get approved. You should only merge changes that are approved.
# OPTIONAL BUT RECOMMENDED
- [ ] Please see inline comments.
## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**